src/utils/index.js

```diff
@@ -9,6 +9,7 @@ module.exports.checkHeadsetConnected = require('./checkHeadsetConnected');
 module.exports.debug = require('./debug');
 module.exports.entity = require('./entity');
 module.exports.forceCanvasResizeSafariMobile = require('./forceCanvasResizeSafariMobile');
+module.exports.functions = require('./functions');
```

Comment:
They'd be utility related to wrapping functions. I could imagine we'd add more function-related utilities in the future (e.g., debounce), and it'd be more organized and consistent to namespace them.